api/src/game/services/role/role.service.ts

```
@@ -0,0 +1,107 @@
+import * as path from 'path';
+import { GameRole, RoleSchema } from 'src/roles';
+
+@Injectable()
+export class RoleService implements OnModuleInit {
+  private roles: Map<string, GameRole> = new Map(); // Format: { [roleId]: role }
+
+  async onModuleInit() {
+    await this.loadRoles();
+  }
+  validateRole(role: unknown): GameRole {
+    debugger;
```

Biome (1.9.4)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)